package-lock.json

```diff
@@ -4438,11 +4018,35 @@
     },
     "node_modules/zod": {
       "version": "3.22.4",
-      "resolved": "https://registry.npmjs.org/zod/-/zod-3.22.4.tgz",
-      "integrity": "sha512-iC+8Io04lddc+mVqQ9AZ7OQ2MrUKGN+oIQyq1vemgt46jwCwLfhq7/pwnBnNXXXZb8VTVLKwp9EDkx+ryxIWmg==",
+      "license": "MIT",
       "funding": {
         "url": "https://github.com/sponsors/colinhacks"
       }
+    },
+    "packages/convex-helpers": {
+      "version": "0.1.24",
+      "license": "MIT",
+      "devDependencies": {
+        "chokidar-cli": "^3.0.0",
+        "typescript": "~5.3"
+      },
+      "peerDependencies": {
+        "convex": "^1.10.0",
+        "hono": "^4.0.5",
+        "react": "^17.0.2 || ^18.0.0",
+        "zod": "^3.22.4"
+      },
+      "peerDependenciesMeta": {
+        "hono": {
+          "optional": true
+        },
+        "react": {
+          "optional": true
+        },
+        "zod": {
+          "optional": true
+        }
+      }
```

Comment:
delete this one if it gets added - it's redundant when you install via path, which the publish script does